### DISCUSSION TITLE
Commits are not showing up on contribution panel

### DISCUSSION THREAD

**Discussion by claudiabg-c** [community/community#39579] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hello, 

I have an issue with my projects' commits. In July I got a new computer and since then I have been making continuous commits to my GitHub repositories. This commits show up on the website just as they should. But the only problem is that in the "Your Contributions" section they don't show, only if it's a new project they show a single green square.
I checked on my VSC and the account I'm using is the one for github and I even tried the "git config --global user.email" on my VSC and my CMD. Everything seems correct. This happens with public and private projects.

Did anyone had the same issue and knows how to solve it?

**Comment by airtower-luna** [comment, marked as answer]
Looking at some of your most recent commits (e.g. [this one](https://github.com/claudiabg-c/Anime-search-engine-Javascript/commit/be3e80cf1d77f3a5b6d0e0791eefa8d218ade083.patch)) the `user.email` does not seem right. In that one it's not even a valid email address, it looks like it's your GitHub username instead. Commits will only be connected to your account if it's an address that's connected to your GitHub account, or the GitHub-provided noreply address if you want to keep your email private. See: [Set commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)

If you want to fix already existing commits you can [rewrite the repository history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History), but that may cause trouble if you've already shared the history with others.

**Reply by claudiabg-c** [reply]
Wow, thank you so much!

This had been bothering me for some months now and you just found the solution easily. I fixed the email commit address, tried it again and it worked!

**Comment by ujiroexotic** [comment]
i have been push to github but my commit is not showing on the contribution graph.

**Comment by devkayes** [comment]
For my case everything ok, but I'm still don't show my commits history in "Your Contributions" sections. @github-staff
